server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationSent.test.ts

```diff
@@ -18,42 +26,83 @@ describe('SufficientInformationSent', () => {
-        .mockReturnValueOnce(user.name)
-        .mockReturnValueOnce(user.email)
-        .mockReturnValue(user.telephoneNumber)
-
-      const page = new SufficientInformationSent({}, assessment)
-
-      expect(page.body).toEqual(body)
-      expect(page.userName).toEqual(user.name)
-      expect(page.emailAddress).toEqual(user.email)
-      expect(page.phoneNumber).toEqual(user.telephoneNumber)
-    })
-    it("should get the fall back applicant's details", () => {
-      const body = {}
-      ;(retrieveOptionalQuestionResponseFromFormArtifact as jest.Mock)
-        .mockReturnValueOnce(undefined)
-        .mockReturnValueOnce(undefined)
-        .mockReturnValue(undefined)
-
-      const page = new SufficientInformationSent({}, assessment)
 
-      expect(page.body).toEqual(body)
-      expect(page.userName).toEqual('')
-      expect(page.emailAddress).toEqual('')
-      expect(page.phoneNumber).toEqual('')
+    describe('if there are not user details on the application', () => {
+      const applicantDetails = applicationUserDetailsFactory.build()
+      const legacyAsssessmentWithoutUserDetails = assessmentFactory.build({
+        application: {
+          caseManagerIsNotApplicant: undefined,
+          caseManagerUserDetails: undefined,
+          applicantUserDetails: undefined,
+        },
+      })
+
+      describe('if the applicant is the case manager', () => {
+        it('should set the the case manager details on the page with details from the form data', () => {
+          when(retrieveOptionalQuestionResponseFromFormArtifact)
+            .calledWith(
+              legacyAsssessmentWithoutUserDetails.application,
+              ConfirmYourDetails,
+              'caseManagementResponsibility',
+            )
+            .mockReturnValue('yes')
+
+          when(userDetailsFromConfirmYourDetailsPage)
+            .calledWith(legacyAsssessmentWithoutUserDetails.application)
+            .mockReturnValue(applicantDetails)
+
+          const page = new SufficientInformationSent({}, legacyAsssessmentWithoutUserDetails)
+          expect(page.caseManager).toEqual(applicantDetails)
+        })
+      })
+
+      describe('if the applicant is not the case manager', () => {
+        it('should set the the case manager details on the page with details from the form data', () => {
+          when(retrieveOptionalQuestionResponseFromFormArtifact)
+            .calledWith(
+              legacyAsssessmentWithoutUserDetails.application,
+              ConfirmYourDetails,
+              'caseManagementResponsibility',
+            )
+            .mockReturnValue('no')
+
+          when(userDetailsFromCaseManagerPage)
+            .calledWith(legacyAsssessmentWithoutUserDetails.application)
+            .mockReturnValue(applicantDetails)
+
+          const page = new SufficientInformationSent({}, legacyAsssessmentWithoutUserDetails)
+
+          expect(page.caseManager).toEqual(applicantDetails)
```

Comment:
This is a little confusing to read given the applicant is not the case manager here. I know it's the same type, but is it worth creating a separate variable with case manager details for clarity?